policy/modules/system/miscfiles.fc

```diff
@@ -10,10 +10,13 @@ ifdef(`distro_gentoo',`
 #
 /etc/avahi/etc/localtime --	gen_context(system_u:object_r:locale_t,s0)
 /etc/httpd/alias/[^/]*\.db(\.[^/]*)* -- gen_context(system_u:object_r:cert_t,s0)
+/etc/httpd/conf/ssl(/.*)?	--	gen_context(system_u:object_r:tls_privkey_t,s0)
```

Comment:
Maybe this should be a new type, since it's httpd's private key, not a general/generic one.

Reply:
Yes, it could be a new type, but it won't add any gain in terms of security at this time.

Consider, the HTTP daemon is the main consumer of TLS/SSL Certificates and corresponding Private Keys. The second main consumer is the Mail Transport daemon. Then possibly the LDAP daemon.

I think at the moment we should focus on fixing the Information Disclosure vulnerability in the shortest time possible, it can be later improved if needed.